class/defaults.yml

```
@@ -1,4 +1,41 @@
 parameters:
   openshift4_splunk_forwarder:
     =_metadata: {}
-    namespace: syn-openshift4-splunk-forwarder
+    multi_instance: true
+    namespace: openshift-logging
+
+    fluentd:
+      loglevel: info
+      replicas: 2
+
+      sharedkey: splunkforwarding
+
+      ssl:
+        enabled: false
+        cert: ""
+        key: ""
```

The SSL private key is a secret so it **MUST** be stored in Vault.

```suggestion
        key: ?{vaultkv:${cluster:tenant}/${cluster:name}/openshift4-splunk-forwarder/${_instance}/fluentd/ssl-key
```